lib/watir/elements/element.rb

```diff
@@ -468,6 +468,13 @@ def browser
     end
 
     #
+    # Delegates script execution to browser or frame.
+    #
+    def execute_script(script, *args)
```

Comment:
I don't follow, why do we introduce `Element#execute_script`? Does it make any sense?